sdk/webpubsub/Azure.Messaging.WebPubSub.Client/README.md

```diff
@@ -67,7 +78,11 @@ A connection, also known as a client connection, represents an individual WebSoc
 
 ### Recovery
 
-If using reliable protocols, a new WebSocket will try to reconnect using the connection ID of the lost connection. If the WebSocket connection is successfully connected, the connection is recovered. And all group contexts will be recovered, and unreceived messages will be resent.
+If using reliable protocols, a new WebSocket tries to establish using the connection ID of the lost connection. If the new WebSocket connection is successfully connected, the connection is recovered. And all group contexts will be recovered, and unreceived messages will be resent. If the service returns WebSocket error code `1008` or the recovery attemption lasts more than 30 seconds, the recovery fails.
```

Comment:
"unreceived messages will be resent" => give it a clearer statement? who unreceived who resend?